backend/tests/test_rate_limit_fix.py

```
@@ -0,0 +1,926 @@
+            "Different tenants should have different limits"
+        
+        # Neither should be hardcoded 1000
+        assert rate_limit1.limit_value != 1000, "Should not use hardcoded 1000"
+        assert rate_limit2.limit_value != 1000, "Should not use hardcoded 1000"
+
+    def test_increment_rate_limit_signature(self):
+        """Test that increment_rate_limit accepts limit_value parameter"""
+        # This test verifies the method signature
+        def increment_rate_limit(tenant_id: UUID, limit_type: str, limit_value: int = 1000) -> None:
+            """Mock implementation"""
+            pass
```

Ruff (0.14.7)

Unnecessary `pass` statement

Remove unnecessary `pass`

(PIE790)